services/ui-src/src/components/ComponentId.tsx

```diff
@@ -69,7 +69,7 @@ const ComponentId: React.FC<{
           <input
             className="field"
             type="text"
-            id={id}
+            id={idPrefix + "componentId"}
             name="componentId"
             aria-describedby={idFieldHint
```

Comment:
Should add the idPrefix to the described-by field as well - currently the form has both of them with fieldhint0...

Reply:
fixed!